### PR TITLE
Optimize random probes API

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: glam/api/views.py [hunk 1]
```diff
@@ -329,43 +329,32 @@ def probes(request):
...
-                )
-            except NotFound:
-                continue
-            if aggregations:
-                probes.append({"data": aggregations, "info": probe.info})
-            if n <= len(probes):
-                break
+    # Get a random list of 9 probes from the Desktop nightly table.
+    aggs = DesktopNightlyAggregationView.objects.raw("""
+        SELECT id, metric, histogram
```
Comment (hamilton) [inline glam/api/views.py:345]: Do we not get total clients here?

Comment (hamilton) [reply glam/api/views.py:345]: probably below, not here, but I am not getting all the info I got before from this.

File: glam/api/views.py [hunk 2]
```diff
@@ -329,43 +329,32 @@ def probes(request):
...
+        LIMIT %s
+    """, [n])
+
+    for agg in aggs:
+        try:
+            probe = Probe.objects.get(info__name=agg.metric)
+        except Probe.DoesNotExist:
+            pass
+
+        probes.append({"data": agg.histogram, "info": probe.info})
```
Comment (hamilton) [inline glam/api/views.py:362]: seems like maybe here is where we'd put the other metadata llike total client counts

Comment (hamilton) [reply glam/api/views.py:362]: Actually, nevermind. I'm just going to get rid of this part.

File: glam/api/views.py (hunk 1 above)
Comment (robhudson) [reply glam/api/views.py:345]: @hamilton Yeah it's not all the same data. Let me know what else you'd like and we can add it back. That was part of the process I thought we'd go over. Basically I was trying to streamline it based on what I see on the home page.

**Pull request merged**